.cirrus.yml

```diff
@@ -30,17 +30,18 @@ env:
     PRIOR_UBUNTU_NAME: "ubuntu-2010"
 
     # Google-cloud VM Images
+    IMAGE_SUFFIX_UBUNTU: "c5521575421149184"
```

Comment:
This could lead to trouble (and maintenance headaches).  Could you add a comment explaining why the same set of images can't be used for both Fedora and Ubuntu?

Also, the `meta_task` will need updating to block the Ubuntu images from being pruned.